src/services/listener.js

```diff
@@ -1,3 +1,33 @@
 // TODO: Implement simple http listener for receiving data from parser
+import express from 'express';
+// import automate from './automator.js';
 
-export {}
+// This is the router from express
+const router = express.Router();
+
+// Define POST method request
+router.post('/', (req, res)=>{
+    const log = req.body;
+    try {
+        // Assume the body structure will be 
+        // {"type": "", "from": "", "timestamp": ""}
+        const type = log.type;
+        const from = log.from;
+        const timestamp = log.timestamp;
+        
+        // Check if the body from request has data that we need 
+        if(type && from && timestamp){
+
+            // Call automator and pass the parameter
+            // automate(type, from, timestamp);
+            res.send({message:'Thanks!'});
+        }else{
+            res.send({message:'The body is not meet the requirement'})
```

Comment:
@basthan untuk respon else bisa dibuat res.status(400).({ message: \`[${payload_yang_tidak_ada}] is required.\` }) gak bas? Untuk message misal gak ada type dan from jadi \`[type, from] is required\`.

Reply:
sudah ditambahkan dicommit selanjutnya